buildbot/osuosl/master/config/builders.py

```diff
@@ -1883,10 +1885,8 @@
                             "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache",
                             "-DLIBOMPTARGET_FOUND_AMDGPU_GPU=ON",
                             "-DLIBOMP_ARCHER_SUPPORT=OFF",
-                            "-DLIBC_GPU_BUILD=ON",
-                            "-DLIBC_GPU_ARCHITECTURES=gfx906",
-                            "-DLIBC_GPU_TEST_ARCHITECTURE=gfx906",
-                            "-DLIBC_GPU_TEST_JOBS=8",
+                            "-DRUNTIMES_amdgcn-amd-amdhsa_LLVM_ENABLE_RUNTIMES=libc",
+                            "-DLLVM_RUNTIME_TARGETS='default,amdgcn-amd-amdhsa'",
```

Comment:
Don't think these are necessary, the Python likely handles the qoutes for us since it does stuff like `"-DLLVM_TARGETS_TO_BUILD=X86;AMDGPU",` above.